src/guides/v2.4/release-notes/commerce-2-4-2.md

```diff
@@ -153,7 +151,7 @@ This release includes enhancements to Page Builder content migration and Page Bu
 
 *  The new mobile [viewport switcher](https://docs-beta.magento.com/user-guide/cms/page-builder-workspace.html#viewports) and viewport scope for form field values lets users and developers perform these actions:
 
-   *  View content on different viewports when authoring.
+   *  View content on different view ports when authoring.
```

Comment:
Original wording is correct

Reply:
I see there's a lot of separation of commonly accepted compound words in this PR.  I don't think it's a big deal for search engines.